pkgs/by-name/pu/pushup/package.nix

```diff
@@ -0,0 +1,34 @@
+{ fetchFromGitHub, buildGoModule, lib, pkgs }:
+buildGoModule rec {
+  pname = "pushup";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "adhocteam";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-9ENXeVON2/Bt8oXnyVw+Vl0bPVPP7iFSyhxwc091ZIs=";
+  };
+
+  vendorHash = null;
+  subPackages = ".";
+  CGO_ENABLED = 0;
```

Comment:
what is CGO used for/why are we disabling it?

Reply:
Pushup does not use CGO so it's common to disable it explicitly for builds.